app/alembic/versions/47da74065fe7_add_student_model.py

```diff
@@ -0,0 +1,38 @@
+"""Add student model
+
+Revision ID: 47da74065fe7
+Revises: 9f4b569fbad6
+Create Date: 2021-03-25 16:59:30.355952
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '47da74065fe7'
+down_revision = '9f4b569fbad6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('student',
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('term_id', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['term_id'], ['term.id'], ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['user_id'], ['user.id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('user_id')
+    )
+    op.create_index(op.f('ix_student_term_id'), 'student', ['term_id'], unique=False)
+    op.create_index(op.f('ix_student_user_id'), 'student', ['user_id'], unique=False)
```

Comment:
Uhh how is unique=False if it's Primary key ;_;

Reply:
No clue

Reply:
Lel